tests/conftest.py

```diff
@@ -2,10 +2,13 @@
 
 import sys
 import pytest
+import os
 from os.path import abspath, dirname, join
 import torch
 import warnings
 
+os.environ['PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION'] = 'python'
```

Comment:
Can we add a comment above this line indicating for which test this is needed?

Reply:
Added a comment describing that this is needed for the T5 unittest.